jobs/influxdb/templates/bin/influxdb_ctl.erb

```diff
@@ -19,12 +19,18 @@ exec 2> >( tee -a /var/vcap/sys/log/influxdb_ctl.err.log | logger -p user.error
 ulimit -c unlimited
 ulimit -n <%= p("influxdb.max_open_fd") %>
 
-for DIR in $DATA_DIR $STORE_DIR $LOG_DIR $RUN_DIR; do
```

Comment:
Looks like now it is going not only no `chown` but also no `mkdir` and no `chmod`.
Is it expected?